### DISCUSSION TITLE
Need to understand how much CPU pressure is applied in % if below configuration is given.

### DISCUSSION THREAD

**Discussion by payalcha** [chaos-mesh/chaos-mesh#4414] [Q&A]
https://chaos-mesh.org/docs/simulate-heavy-stress-on-kubernetes/#cpustressor 
workers - 10 
load - 50 
What it means. I need to understand in % how much my CPU stress is applied.

**Comment by tobiasjongen** [comment]
Here is how I understand the docs:

> The final sum of CPU load is workers * load.

So in your case, the load would be 10 * 50 = 500. If you now want to now the load of your CPU in percent, you have to divide the value by the number of threads your CPU has. If we assume that your CPU has 4 cores and supports multithreading, you would have 8 threads. This means that the load applied on your CPU woudl be 500/8 = 62.5%.
